src/cgtop/cgtop.c

```diff
@@ -672,7 +673,7 @@ static void help(void) {
                "  -n --iterations=N   Run for N iterations before exiting\n"
                "  -b --batch          Run in batch mode, accepting no input\n"
                "     --depth=DEPTH    Maximum traversal depth (default: %u)\n"
-               "  -M --machine=       Show container\n"
+               "  -M --machine=       Show container (incompatible when cgroup path is specified)\n"
```

Comment:
It could have sense, if you want to monitor different subtrees with this patch you have to select the first common root. With multiple roots you can specify only which one you want to monitor and "cut" the branches you do not care about form the output.

example: you have the same set of services on different machines (e.g. apache and cassandra), but you do not want monitor only the resources used my all the instances of apaches across the different machines.

Also I am expecting that this features is going to be used a lot so it make sense to have it like ti is the "default" behaviour.
However, if you think that it is better with a switch, it is not a problem. I will fix it in a jiffy.